lib/oppen/token.rb

```diff
@@ -1,31 +1,43 @@
 # frozen_string_literal: true
 
-# Oppen
+# Oppen.
 module Oppen
-  # Token used in the oppen algorithm.
+  # Token module.
   module Token
+    # BreakType.
+    #
+    # FITS => No break is needed (the block fits on the line)
+    # INCONSISTENT => New line will be forced only if necessary
+    # CONSISTENT => Each subblock of the block will be placed on a new line
```

Comment:
Still not enough. Open an issue and we'll revisit later.
We should put ourselves in the mindset of a newcomer who barely knows how Oppen works, so our explanation needs to be:

1. Self-contained
2. Complete
3. Leaves no room for doubts.